### DISCUSSION TITLE
T-Beam Helium Mapper to Meshtastic...

### DISCUSSION THREAD

**Discussion by N2YQO** [meshtastic/Discussions#66] [ESP32]
I need help.. I have been trying to locate help from Max_Plastix.. I used to do helium mapping with my T-Beam, but wanted to use it now for meshtastic. Like his issues, my gps and satellites are no go after swapping to meshtastic. After flashing, I receive this: WARN  | ??:??:?? 7 [GPS] No GNSS Module (baudrate 9600)... WARN  | ??:??:?? 16 [GPS] GPS FactoryReset requested...
I flashed with Meshtastic 2.5.11.8e2a3e5 Beta. 

I am referring to Max_Plastix issues linked at https://meshtastic.discourse.group/t/no-gps-module-in-t-beam-v1-1-w-2-1-17-7-alpha/7842

Thank you

**Comment by rcarteraz** [comment]
> I need help.. I have been trying to locate help from Max_Plastix.. I used to do helium mapping with my T-Beam, but wanted to use it now for meshtastic. Like his issues, my gps and satellites are no go after swapping to meshtastic. After flashing, I receive this: WARN | ??:??:?? 7 [GPS] No GNSS Module (baudrate 9600)... WARN | ??:??:?? 16 [GPS] GPS FactoryReset requested... I flashed with Meshtastic 2.5.11.8e2a3e5 Beta.
> 
> I am referring to Max_Plastix issues linked at https://meshtastic.discourse.group/t/no-gps-module-in-t-beam-v1-1-w-2-1-17-7-alpha/7842
> 
> Thank you

Did you do a full wipe and install? Also, the first time you boot up the device with Meshtastic it'll take a good 30 mins for it update the gps and get a lock. It's best to have it outside with clear line of sight of the sky.

**Reply by N2YQO** [reply]
I did do a full wipe and install. I’ve had it running for over a day now too, open view of the sky but I get no satellites.

**Comment by fifieldt** [comment]
Do you happen to have logs?
